### PR TITLE
Userstory7

### PR DESCRIPTION
**Pull request by kisa5**

### CODE REVIEW

File: components/ProjectList.js [hunk 1]
```diff
@@ -2,27 +2,77 @@ import styled from "styled-components";
...
+  word-wrap: break-word;
+  word-break: break-all;
+  border: none;
+  font-size: 14px;
+  color: #696969;
+  box-shadow: 0 4px 14px 0 rgb(0 0 0 / 10%);
+`;
+
 const Wrapper = styled.div`
   margin: 50px 0px 55px 0px;
```
Comment (SaschaRadojkovic) [inline components/ProjectList.js]: you should remove the px when there is a 0

**Review by SaschaRadojkovic — CHANGES_REQUESTED** [review]
only one little thing , everything else looks good to me

File: components/Modal.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import ReactDOM from "react-dom";
+
+export default function Modal({ appear, children }) {
+  const modalContent = appear ? <StyledModal>{children}</StyledModal> : null;
```
Comment (marcterre) [inline components/Modal.js:5]: you could also write ```modalContent = appear ? <..>...</...> : null``` like this ```modalContent = appear && <...>...</...>```
its easier to read and does the same

File: components/ProjectList.js [hunk 3]
```diff
@@ -2,29 +2,79 @@ import styled from "styled-components";
 import Link from "next/link";
 import { useRouter } from "next/router";
 import { useState } from "react";
+import dynamic from "next/dynamic";
+
+const ModalDeleteProject = dynamic(
```
Comment (marcterre) [inline components/ProjectList.js:7]: nice

File: components/ProjectList.js [hunk 4]
```diff
@@ -2,29 +2,79 @@ import styled from "styled-components";
...
     </Wrapper>
   );
 }
 
+const SearchBar = styled.input`
+  width: 90%;
+  margin: 5px;
+  padding: 10px;
+  border-radius: 7px;
+  border: 1px solid grey;
```
Comment (marcterre) [inline components/ProjectList.js:66]: i guess you will style your project in the next days but try to use responsive units consistently like em, rem

**Review by marcterre — APPROVED** [review]
LGTM, left just a small recommendation 🙂
Have fun ✌️

File: components/ProjectList.js (hunk 4 above)
Comment (kisa5) [reply components/ProjectList.js:66]: yeah this is coming up next week ^^

**Pull request merged**